app/assets/stylesheets/spotlight/_breadcrumbs.scss

```diff
@@ -1,8 +1,10 @@
 .breadcrumb-item {
```

Comment:
I believe `breadcrumb-item` comes from bootstrap, so I think we should avoid overriding the upstream styling.

Reply:
@cbeer Ok makes sense, thanks for reviewing! I added the specific class `truncated-value` back into the code. It is added at the same level as `breadcrumb-item` however, rather than to the child span like before, so that the styling will work.

Reply:
I will submit a DLME PR for that other issue since I already wrote the code